src/auro-calendar-month.js

```diff
@@ -58,6 +70,61 @@ export class AuroCalendarMonth extends RangeDatepickerCalendar {
     this.dayNamesOfTheWeek = newDayNamesOfTheWeek;
   }
 
+  parseDateContentByDay() {
+    this.dateSlotContent = [...this.querySelectorAll('[slot^="date_"]')];
+
+    if (this.dateSlotContent && this.dateSlotContent.length > 0) {
+      const items = [];
+
+      this.dateSlotContent.forEach((content) => {
+        const date = new Date(content.getAttribute('date'));
+
+        items.push({
+          date,
+          content
+        });
+      });
+
+      this.slotContentByDay = _.groupBy(items, ({date}) => date.getDate()); // eslint-disable-line no-undef
+    }
+
+    this.insertSlotContentByDay();
+  }
+
+  insertSlotContentByDay() {
+    const renderedDays = [...this.shadowRoot.querySelectorAll('auro-calendar-cell')];
+
+    renderedDays.forEach((dateCell) => {
+      const day = new Date(dateCell.day.date * 1000).getDate();
+      const dayName = this.getFormattedDate(dateCell);
+
+      if (this.slotContentByDay && this.slotContentByDay[day] && this.slotContentByDay[day][0].content.getAttribute('date') === dayName) {
```

Comment:
This `if` statement is pretty procedural. Could this be optimized to be a function instead?